docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -1,31 +1,31 @@
 # Importing an existing volume
 
-Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) yaml file.
+Use this information to import volumes created externally from the IBM® block storage CSI driver by using a persistent volume (PV) YAML file.
 
-Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) yaml file:
+Before starting to import an existing volume, find the following information in the existing volume in order to include the information in the persistent volume (PV) YAML file:
+- `volumeHandle`
+- `volumeAttributes` (optional)
+  
+  Including:
 
--   `volumeHandle`
--   `volumeAttributes` (optional)
-
-    Including:
-
-    -   `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
-    -   `storage_type`: <SVC | A9K | DS8K>
-    -   `volume_name`: _Volume name_
-    -   `array_address`: _Array address_
+    - `pool_name`: _Name of Pool where volume is located_ (Listed as `pool_id` for DS8000® Family systems.)
+    - `storage_type`: <`SVC` | `A9K` | `DS8K`>
+    - `volume_name`: _Volume name_
+    - `array_address`: _Array address_
 
 To find the `volumeHandle`, use one of the following procedures:
 
-- Through command line (for Spectrum Virtualize Family):
+
+- **Through command line (for Spectrum Virtualize Family):**
 
   `lsvdisk <volume name> | grep vdisk_UID`
```

Comment:
shouldn't we link to the command docs, instead of maintaining our own examples (as we did with [lspartnership](https://github.com/IBM/ibm-block-csi-driver/blob/8d18ae5a6d95b15894c814e59a27c133aea06d8a/docs/content/configuration/csi_ug_config_replication_find_systemid.md))?

Reply:
DONE - @roysahar-ibm  @barakda  @tyichye19  Please approve as well for QA.
for reference - https://www.ibm.com/docs/en/flashsystem-9x00/8.4.x?topic=commands-lsvdisk

also updated the A9000/R CLI section - please approve - for reference: https://www.ibm.com/docs/en/flashsystem-a9000r/12.3.2?topic=commands-listing-volumes-extended-attributes

Also, is this applicable for DS8K? if so - where would this be found? does this section help? https://www.ibm.com/docs/en/ds8900/9.2.0?topic=commands-volume-group-specific

Reply:
Looks ok, I will let @roysahar-ibm to say his word on this too

Reply:
OK by me.